package.json

```diff
@@ -32,7 +32,7 @@
   },
   "homepage": "https://github.com/revdotcom/revai-node-sdk#readme",
   "dependencies": {
-    "axios": "^0.18.0",
+    "axios": "^0.19.0",
```

Comment:
?

Reply:
I'm not sure what's going on with this. Develop definitely already has these changes. I think it may be because I changed the default branch on github